homeassistant/components/wallbox/number.py

```diff
@@ -26,15 +28,65 @@
 from .entity import WallboxEntity
 
 
+def min_charging_current_value(coordinator: WallboxCoordinator) -> float:
+    """Return the minimum available value for charging current."""
+    min_value = (
+        (coordinator.data[CHARGER_MAX_AVAILABLE_POWER_KEY] * -1)
+        if coordinator.data[CHARGER_DATA_KEY][CHARGER_PART_NUMBER_KEY][0:2]
+        in BIDIRECTIONAL_MODEL_PREFIXES
+        else 0
+    )
```

Comment:
```suggestion
    min_value = 0
    if coordinator.data[CHARGER_DATA_KEY][CHARGER_PART_NUMBER_KEY][0:2] in BIDIRECTIONAL_MODEL_PREFIXES:
        min_value = coordinator.data[CHARGER_MAX_AVAILABLE_POWER_KEY] * -1
```

Reply:
I made it slightly different now (also wasn't content with the first attempt), can you have another look